app/com/arpnetworking/metrics/portal/scheduling/impl/NeverSchedule.java

```diff
@@ -19,7 +19,7 @@
 import java.util.Optional;
 
 /**
- * Schedule that should never fire.
+ * BaseScheduleViewModel that should never fire.
```

Comment:
Refactoring error?

Reply:
Yes (to this and all similar changes) - reverted.